### DISCUSSION TITLE
Update to Italian locale [it]

### DISCUSSION THREAD

**Discussion by Night-Train-666** [aclap-dev/video-downloadhelper#122] [Language & Translation]
I have updated the italian translation of missing strings for 8.2.0.27a1 (beta) version.
This is the continuation of the official thread for the Italian translation started here:
bye.
[missing-strings-8.2.0.27a1-beta.json](https://github.com/aclap-dev/video-downloadhelper/files/14313964/missing-strings-8.2.0.27a1-beta.json)

**Comment by paulrouget** [comment]
Thank you! I will add that to the upcoming version.

**Comment by Night-Train-666** [comment]
Hi Paul,

I have updated the italian translation of missing strings for 9.0.1.9a1 (beta) version, but the tab relating to the strings to be translated does not appear (I searched for them manually).

Unfortunately, however, the entries in English continue to appear and the Italian is not taken into consideration despite having saved them and reloaded the addon.

Furthermore, using the new UI, it is not possible to choose the translated string file to import (the file browser does not open).

I hope I have been helpful.

Thanks to everyone who collaborates on this fantastic addon!

bye

Night Train

[missing-strings-it-9-0-1-9a1-beta-messages.json](https://github.com/aclap-dev/video-downloadhelper/files/15265111/missing-strings-it-9-0-1-9a1-beta-messages.json)

**Comment by paulrouget** [comment]
@Night-Train-666 I'm not entirely sure to understand the behavior you are describing. Further more, you are sharing a JSON file. Version 9 doesn't export as JSON, but as a .TXT. So I'm a little confused about what exactly is going on.

Can you share screenshot? That would help!

Thanks a lot for your help :)

**Reply by Night-Train-666** [reply]
Hi Paul,
I checked, in fact if I use the new UI I export a txt file, if I use the legacy UI then I export a json file.
I have updated the italian translation of missing strings for 9.0.2.3a1 (beta) version.
bye

Night Train

[it.txt](https://github.com/user-attachments/files/15519397/it.txt)

**Comment by franzil** [comment]
[it.txt](https://github.com/aclap-dev/video-downloadhelper/files/15413999/it.txt)

Italian translation for version 9

Cheers

**Comment by Giack271** [comment]
Hello,

this morning I updated the missing italian translation too for _Version 9.0.1.13 (stable), target mozilla, locale it_.

Please find the file attached here: [it.txt](https://github.com/aclap-dev/video-downloadhelper/files/15416591/it.txt)

Kind regards,

Giacomo

**Comment by Night-Train-666** [comment]
Hi Paul,

I have updated the italian translation of missing strings for 9.0.2.12a1 (beta) version.
it.txt

I also attach a clean and correct version of the it.toml file which I downloaded on GitHub.
(renamed .txt to be able to attach)

In this regard, to avoid confusion and the risk of having strings translated with disparities in linguistic knowledge or interpretation of commands, perhaps it would be better to indicate an official thread for that language and put the latest available file at the beginning of the thread with the date and the version.

I noticed a few things:

- in tab Translation there is not option "import" (only export);

- some entries are extra and others no longer exist or have double names (with v9_ in front);

- the extension popup shows:

There are 5 strings not translated in your language (it). You can help us by translating or improving the strings: open translation panel.

which should be translated into:

Ci sono 5 stringhe non tradotte nella tua lingua (it). Puoi aiutarci traducendo o migliorando le stringhe: apri il pannello di traduzione.

[new-it.toml.txt](https://github.com/user-attachments/files/15889159/new-it.toml.txt)
[version-9.0.2.12a1-(beta)-it.txt](https://github.com/user-attachments/files/15889162/version-9.0.2.12a1-.beta.-it.txt)

**Comment by BondiBo** [comment]
[it.txt](https://github.com/user-attachments/files/15996290/it.txt)
Just now 2024/06/26 I have corrected and completed the translation of all visible text in the Italian language (see attached file it.txt)

**Comment by paulrouget** [comment]
These changes will be reflected soon.

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
